tests/botstarter/test_common.py

```diff
@@ -1,8 +1,15 @@
-from botstarter.common import unwrap_answer
+from botstarter.common import *
 
 
-def test_unwrap_answer():
-    user_input = "—opt"
+def test_str2mdown():
+    str = "-!.>_"
 
-    result = unwrap_answer(user_input)
-    assert "--opt" == result, "User input was not unwrapped correctly."
+    result = str2mdown(str)
+    assert "\\-\\!\\.\\>\\_" == result, "Str was not converted to mdown correctly."
+
+
+def test_empty_input_str2mdown():
```

Comment:
Hi @ceccatoandrea, regarding this test, if a `None` value is passed to the function it will just throw a NullPointer because there is no null-check on the `text` parameter.

Could you please add a null check to the `str2mdown` function so it returns None if a None parameter is passed?